### PR TITLE
Add DataGenerator

### PR DESCRIPTION
**Pull request by tiagojsag**
Add DataGenerator

### CODE REVIEW

File: src/Extensions/Shopware/DataGenerator/Resources/Customers.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            $id = $this->getUniqueId('customer');
+            $customerNumber = $this->getUniqueId('customerNumber');
+
+            $group = $customerCounter % 2 === 1 ? "EK" : "H";
+            $sex = $customerCounter % 2 === 1 ? "mr" : "ms";
+
+            $import_customers->write(
+                "{$id},a256a310bc1e5db755fd392c524028a8,user_{$id}@example.org,1,0,,5,2013-01-11,2015-01-01 00:00:00,,0,,0,{$group},0,1,1,,\N,,0,\N"
+            );
+            $import_customers_attributes->write("{$id}");
```
Comment (bcremer) [inline src/Extensions/Shopware/DataGenerator/Resources/Customers.php]: Please fix the remaining CS issues.

**Comment by bcremer** [comment]
Looks fine for me. Please fix the remaining CS issues and it's good for merge.

@dnoegel Your opinion on that one?

**Comment by tiagojsag** [comment]
CS fixed

**Comment by bcremer** [comment]
Can you also add the new command to the README please. Thanks.

**Comment by tiagojsag** [comment]
I'd rather remove all command-specific docs from the README and spend some time improving each command's description and the args/options help texts. What do you think?

**Pull request merged**